Code/DataStructs/Wrap/testSparseIntVect.py

```diff
@@ -212,6 +212,19 @@ def test6BulkTversky(self):
     for i in range(len(bulkDs)):
       self.assertTrue(feq(bulkDs[i], taniDs[i]))
 
+  def test7ToList(self):
```

Comment:
Since this test is for a `SparseBitVect` not a `SparseIntVect` it would make more sense to include it in `testBV.py` (after the other new test you added).